webapp/src/components/rhs_sidebar/rhs_sidebar.jsx

```diff
@@ -0,0 +1,418 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+
+import MattermostGitLab from './mattermost_gitlab';
+import NoSubscriptions from './no_subscriptions';
+
+const notSignedInStyle = {
+    margin: '24px',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    textAlign: 'center',
+};
```

Comment:
@lieut-data In general, our plugins have been using `scss`. I know Playbooks is using `styled-components` and no `scss` now, though I'd like to keep consistency with the other plugin projects our team manages in this aspect.

Using `css` instead of `scss` can be a pain, because you don't have the same easily accessible nesting capabilities with `css`. Of course there's the equivalent syntax in `css`, which may not be too bad in this case.

I'll leave it up to you to decide how to apply styles here. I'm 2/5 on not introducing `styled-components` here though. Hopefully you can understand why 🙂

Reply:
I'm happy to keep things simpler for the maintainers and will switch to CSS.